feature-libs/my-account/organization/components/shared/organization-detail/toggle-status-action/toggle-status.component.ts

```diff
@@ -94,7 +95,17 @@ export class ToggleStatusComponent<T extends BaseItem> implements OnDestroy {
   }
 
   protected update(item: T): void {
-    this.itemService.update(item[this.key], this.getPatchedItem(item));
+    this.itemService
+      .update(item[this.key], this.getPatchedItem(item))
+      .pipe(
+        take(1),
+        tap((data) => {
```

Comment:
do we want to display here any error message?

Reply:
yes, that would be nice, but I assume most errors might end up in the current global message component. If that's the case, it's fine for now.

Reply:
We don't know a list of possible backend errors / codes so we don't have that in translations. E.g `common:httpHandlers.validationErrors.invalid.budgetData`

Reply:
As discussed we move the conversations regarding errors to new ticket. 

I'd nit about changing this code to: 

```ts
this.itemService
      .update(item[this.key], this.getPatchedItem(item))
      .pipe(
        take(1),
        filter((data) => data.status === LoadStatus.SUCCESS)
      )
      .subscribe((data) => this.notify(data.item));
```